source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,534 @@ Filters Builders
+   { "_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"] }
+   { "_id": 6, "color": "pink", "qty": 5, "vendor": ["C"] }
+   { "_id": 7, "color": "green", "qty": 8,"vendor": ["B", "C"] }
+   { "_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"] }
+
+Collection: ``binary_numbers``
+
+.. code-block:: json
+
+   { "_id": 9, "a": 54, "binaryValue": "00110110" }
+   { "_id": 10, "a": 20, "binaryValue": "00010100" }
+   { "_id": 11, "a": 68, "binaryValue": "1000100" }
+   { "_id": 12, "a": 102, "binaryValue": "01100110" }
+
+Collection: ``geo_points``
+
+.. code-block:: json
+
+   { "_id": 13, "coordinates": { "type": "Point", "coordinates": [2.0, 2.0] } }
+   { "_id": 14, "coordinates": { "type": "Point", "coordinates": [5.0, 6.0] } }
+   { "_id": 15, "coordinates": { "type": "Point", "coordinates": [1.0, 3.0] } }
+   { "_id": 16, "coordinates": { "type": "Point", "coordinates": [4.0, 7.0] } }
+
+.. _comparison:
+
+Comparison 
+----------
+
+The comparison filters include all operators that compare the value in a
+document to a specified value. 
+
+The comparison operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Comparison Method
+     - Matches
+      
+   * - :java-core-api:`eq() <com/mongodb/client/model/Filters.html#eq(java.lang.String,TItem)>`
+     - values equal to a specified value.
+
+   * - :java-core-api:`gt() <com/mongodb/client/model/Filters.html#gt(java.lang.String,TItem)>`
+     - values greater than a specified value.
+      
+   * - :java-core-api:`gte() <com/mongodb/client/model/Filters.html#gte(java.lang.String,TItem)>`
+     - values greater than or equal to a specified value.
+
+   * - :java-core-api:`lt() <com/mongodb/client/model/Filters.html#lt(java.lang.String,TItem)>`
+     - values less than a specified value.
+
+   * - :java-core-api:`lte() <com/mongodb/client/model/Filters.html#lte(java.lang.String,TItem)>`
+     - values less than or equal to a specified value.
+
+   * - :java-core-api:`ne() <com/mongodb/client/model/Filters.html#ne(java.lang.String,TItem)>`
+     - values not equal to a specified value.
+
+   * - :java-core-api:`in() <com/mongodb/client/model/Filters.html#in(java.lang.String,java.lang.Iterable)>`
+     - any of the values specified in an array.
+
+   * - :java-core-api:`nin() <com/mongodb/client/model/Filters.html#nin(java.lang.String,java.lang.Iterable)>`
+     - none of the values specified in an array.
+
+   * - :java-core-api:`empty() <com/mongodb/client/model/Filters.html#empty()>`
+     - all the documents.
+
+The following example creates a filter that matches all documents where
+the value of the ``qty`` field equals "5" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin equalComparison
+   :end-before: end equalComparison
+   
+The JSON output renders as the following:
```

Comment:
I found this a bit confusing, since the code above doesn't actually render the filter as JSON.  Do you think we should show code that renders the filter?

Reply:
I thought it would help clarify to the reader what the filter is doing. This was brought up in another PR for me too and I was told we can assume the user has a general understanding and won't need to see what it renders as.  I'll take these out in all sections.

Reply:
No, I like it and am not saying we should remove.  I was just wondering if we should show a sample of how one would show the rendering oneself.

Reply:
But if you all decide to remove it I'm ok with that too.